track_and_trace/asset_client/src/views/agent_detail.js

```diff
@@ -96,42 +96,6 @@ const editIcon = (obj, key) => {
   return forms.clickIcon('pencil', () => { obj[key] = !obj[key] })
 }
 
-// Edits a field in state
-const editField = (state, label, key) => {
```

Comment:
Can you add a comment about why this stuff was deleted to the commit. Does it belong in a different commit?

Reply:
Updated commit message